ruspy/simulation/simulation_auxiliary.py

```diff
@@ -66,11 +66,13 @@ def simulate_strategy(
             decisions[bus, period] = decision
             utilities[bus, period] = utility
             new_state = intermediate_state + state_increase
+            if new_state > num_states:
```

Comment:
Did you introduce an absorbing state? Could you explain why the error is not necessary any more?